### PR TITLE
Fix bot clean up

### PR DESCRIPTION
**Pull request by EmilyBourne**
The bot is currently failing in the clean-up due to a stupid error. This problem prevents coverage tests being run automatically. This PR fixes it.

Additionally packages were installed on the wrong Python version for anaconda resulting in windows results never being reported. This is also fixed

### CODE REVIEW

**Review by sboof911 — APPROVED** [review]
Looks Good

File: ci_tools/bot_comment_react.py [hunk 1]
```diff
@@ -29,6 +29,9 @@ def get_unique_test_list(keys):
     tests.discard('pr_tests')
     if 'coverage' in tests:
         tests.add('linux')
+        # Ensure coverage is last in case dependencies are ready
+        tests.discard('coverage')
+        tests.add('coverage')
```
Comment (yguclu) [inline ci_tools/bot_comment_react.py:34]: I do not understand this change, as Python sets are not ordered AFAIK. Maybe you want to use a list instead?

**Review by yguclu — CHANGES_REQUESTED** [review]

File: ci_tools/bot_comment_react.py (hunk 1 above)
Comment (EmilyBourne) [reply ci_tools/bot_comment_react.py]: Ah. You are right, dictionaries have confused me. Fixed: a62afc4989a35e521d082320c04c3b808b65f466

File: ci_tools/bot_comment_react.py [hunk 2]
```diff
@@ -27,9 +27,13 @@ def get_unique_test_list(keys):
     if 'pr_tests' in tests:
         tests.update(pr_test_keys)
     tests.discard('pr_tests')
+    result = list(tests)
     if 'coverage' in tests:
-        tests.add('linux')
-    return tests
+        result.append('linux')
```
Comment (yguclu) [inline ci_tools/bot_comment_react.py:32]: We should be careful here when using a list. If `linux` is already in the list you will end up with two identical entries.

**Review by yguclu — CHANGES_REQUESTED** [review]

File: ci_tools/bot_comment_react.py (hunk 2 above)
Comment (EmilyBourne) [reply ci_tools/bot_comment_react.py:32]: Oops

Comment (EmilyBourne) [reply ci_tools/bot_comment_react.py]: Fixed b6d3319b

**Review by yguclu — APPROVED** [review]

**Pull request merged**